improve/CSA/Config/Common.py

```diff
@@ -164,7 +164,8 @@ def initialize_parameters(self,
             os.makedirs(self.output_dir)
     
         self.__class__ = current_class
-        return self.dict()
+        #return self.dict()
+        return self.option
```

Comment:
This is a good workaround but does not return what is needed. The function which returns the config as a dictionary is missing and needs to be implemented or copied.